modal_proto/api.proto

```diff
@@ -1926,6 +1938,11 @@ message ProxyListResponse {
   repeated Proxy proxies = 1;
 }
 
+message PythonPackageVersions {
```

Comment:
j/w but why does this need its own message class?

Reply:
It was to be able to differentiate empty map from missing map, but I just changed that since I realized we store the map as a PGDict anyways on the backend which doesn't differentiate those. Might add a boolean in the future if we want "presence" for the field

Reply:
Got it. I don't think we should ever have an "true" empty map of third-party Python packages given that we should install the client deps in all base images right? Or maybe I'm missing something.